tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,152 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+from tests.helper_functions import get_safe_port
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+pytestmark = pytest.mark.skipif(
+    "UC_OSS_INTEGRATION" not in os.environ,
+    reason="This test is only valid w/in the github workflow integration job",
+)
+
+
+@pytest.fixture(scope="module")
+def setup_servers():
+    port = get_safe_port()
+    with (
+        subprocess.Popen(
+            ["bin/start-uc-server"],
```

Comment:
Does `bin/start-uc-server` have a `--port` option for specifying the port?

Reply:
Unfortunately no.  I believe this is a setting defined in the server config.